src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -564,6 +571,15 @@ parseBinderNoParens = P.choice
                       , ParensInBinder <$> parens parseBinder
                       ] P.<?> "binder"
 
+parseBinderLocalDecl :: TokenParser Binder
```

Comment:
It's because `parseBinderAtom` includes literal binds and null binder as well. I thought it's better not allowing them in parsing and use separate sets of binders.